docs/en/operation/StressBench.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: global
+title: StressBench Framework
+nickname: StressBench
+group: Operations
+priority: 7
+---
+
+* Table of Contents
+  {:toc}
+
+The Alluxio system runs in different environments with different hardware for various purposes. The
+StressBench test suite is a performance benchmark suite to help you understand the performance of
+Alluxio in different scenarios. It will help you quickly test and analyze the Alluxio usage in your
+environment by simply running the command line.
+
+The StressBench is consisted of different tests to help you test the metadata and IO performance of
+the Alluxio system. The following benchmark tools are currently supported:
+* `StressMasterBench` - A benchmark tool measuring the master performance of Alluxio.
+* `StressClientBench` - A benchmark tool measuring the IO performance of Alluxio through the client.
+* `StressWorkerBench` - A benchmark tool measuring the IO performance of Alluxio with single node.
```

Comment:
I feel the description are a little unclear. For example for the "master performance of Alluxio" maybe it's better to mention the metadata operations? Same for the other two commands. Maybe we can further clarify what "through the client" means to a user? Also is `StressWorkerBench` using only a single node (I don't know how this test works but this descriptions makes me want to ask)?